Sources/Subs-Editor.php

```diff
@@ -2464,7 +2464,7 @@ function AutoSuggest_Search_SMFVersions()
 
 	// Just in case we don't have ANYthing.
 	if (empty($versions))
-		$versions = array('SMF 2.0');
+		$versions = array('SMF 2.1');
```

Comment:
It makes more sense to use SMF_FULL_VERSION here. If something has gone wrong with retrieving latest-versions.txt, the one version string we do know for sure is SMF_FULL_VERSION. In contrast, `SMF 2.1` is not a version string that has ever or will ever appear in latest-versions.txt.